src/backend/Functions/ProcessWorkflowPoisonQueue.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using WorkflowManagement;
+
+namespace Functions
```

Comment:
@nicolehaugen79 Since this is merged, you may have to open a PR to pull this into `PortabilityService.Functions/` rather than `Functions/`